playbooks/azure/openshift-cluster/build_node_image.yml

```diff
@@ -0,0 +1,97 @@
+---
+- hosts: localhost
+  gather_facts: no
+  tasks:
+  - name: provision resource group
+    import_tasks: tasks/provision_instance.yml
+    vars:
+      image_prefix: "{{ openshift_azure_input_image_prefix }}"
+      image_resource_group: "{{ openshift_azure_input_image_ns }}"
+      create_args: --data-disk-sizes-gb 128
+
+  - set_fact:
+      openshift_node_bootstrap: True
+
+- hosts: nodes
+  tasks:
+  - set_fact:
+      openshift_deployment_type: "{{ 'openshift-enterprise' if ansible_distribution == 'RedHat' else 'origin' }}"
+      openshift_enable_origin_repo: False
+      openshift_node_bootstrap: True
+      skip_node_svc_handlers: True
+      openshift_master_fire_handlers: False
+      openshift_master_systemd_enable: False
+      openshift_additional_repos: "{{ azure_node_repos[ansible_distribution] }}"
+
+  - set_fact:
+      openshift_additional_repos: "{{ openshift_additional_repos + [{'name': 'install_repo', 'baseurl': openshift_azure_install_repo, 'enabled': true, 'gpgcheck': false}] }}"
+    when: openshift_azure_install_repo is defined and openshift_azure_install_repo
+
+  - name: install centos-release-paas-common rpm
+    yum:
+      name: centos-release-paas-common
+      state: present
+    when: ansible_distribution == "CentOS"
+
+  - name: configure yum repositories
+    import_tasks: tasks/yum_certs.yml
+
+  - name: update rpms
+    import_role:
+      name: os_update_latest
+    vars:
+      os_update_latest_reboot: True
+
+- name: install openshift
+  import_playbook: ../../openshift-node/private/image_prep.yml
+
+- hosts: nodes
+  tasks:
+  - import_role:
+      name: openshift_master_facts
+
+  - name: setup systemd units
+    import_role:
+      name: openshift_master
```

Comment:
This is not needed and will be removed.  It is however used in the 3.9 branch.  @jim-minter, should I remove this in this cherry-pick or should I add a follow on commit to remove, or should we clean up in subsequent PR since this will be the work required to bring up masters on 3.10?

This will update to `roles/openshift_control_plane`

Reply:
remove it in the cherry-pick in this PR.  If there are additions in subsequent PRs that's fine

Reply:
yeah, this will have to be removed for tox to pass